code/__DEFINES/sound.dm

```diff
@@ -9,6 +9,17 @@
 #define CHANNEL_BUZZ 1017
 #define CHANNEL_BICYCLE 1016
 
+///Default range of a sound.
+#define SOUND_RANGE 17
+///default extra range for sounds considered to be quieter
+#define SHORT_RANGE_SOUND_EXTRARANGE -9
+///The range deducted from sound range for things that are considered silent / sneaky
+#define SILENCED_SOUND_EXTRARANGE -11
```

Comment:
What's the difference between these 2 defines?

Reply:
-2,

but all kidding aside, I used both where they sounded best, silenced is basically close to what we currently have for audio, and short range was just slightly further than that.

Reply:
gottcha ~~I didn't see the minus on that 9~~